exercises/practice/rotational-cipher/.approaches/introduction.md

```diff
@@ -5,7 +5,7 @@
 - The key lesson from this exercise is that you can define two methods for the function `rotate`
 - The macro/metaprogramming is quite challenging for lots of students. Feel free to read the answer below!
 
-## Example solution
+## Approach: modulo arithmetic
```

Comment:
Almost any exercise with rotation will use modular arithmetic at some point, so I don't think this is a great name, but it's hard to think of a good one!

```suggestion
## Approach: using `Char` arithmetic and `map`
```

This is also generic, because most solutions will use `map`, but it's a bit more specific, imo.